app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Lobster&display=swap');
+
+:root {
+  --main-color: #3778c2;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: 'Figtree', sans-serif;
+  margin: 0;
+}
+
+body {
+  display: grid;
+  place-items: center;
+  background: whitesmoke;
+}
+
+main {
+  width: 375px;
+  height: 100vh;
```

Comment:
* Kindly ensure you follow the design guidelines provided especially the colors and font-family  
![image](https://github.com/GraceHtet/budget-app/assets/52497246/54dfe663-c099-47a8-a2d4-b9fab499d830)
![image](https://github.com/GraceHtet/budget-app/assets/52497246/5468a03a-1445-46c6-acdf-b1e8917334ab)